<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>小虾米</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/form.png</normaloff>icons/form.png</iconset>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="settings_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>UART Settings</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <item row="1" column="1">
       <widget class="QLineEdit" name="BaudRate_lineEdit">
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="Port_num_comboBox">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="open_pushButton">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>UART Open</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="stop_pushButton">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>UART Stop</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="HOSI_IP_Label">
        <property name="text">
         <string>Baud Rate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="socket_type_label">
        <property name="text">
         <string>Serial Port</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>32</y>
       <width>231</width>
       <height>199</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ACC-X:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="acc_x_Label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ACC-Y:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="acc_y_Label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ACC-Z:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="acc_z_Label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>GYO-X:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="gyo_x_Label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>GYO-Y:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="gyo_y_Label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>GYO-Z:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="gyo_z_Label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Canvas Settings</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>236</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="canvas_pause_pushButton">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Canvas Pause</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="canvas_start_pushButton">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Canvas Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Legend</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>231</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="x_legend_label">
        <property name="font">
         <font>
          <family>Yu Gothic UI Semibold</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>X:----------</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextEditable</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="y_legend_label">
        <property name="font">
         <font>
          <family>Yu Gothic UI Semibold</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>y:----------</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextEditable</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="z_legend_label">
        <property name="font">
         <font>
          <family>Yu Gothic UI Semibold</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Z:----------</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextEditable</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>1301</width>
      <height>731</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Live</string>
     </attribute>
     <widget class="QGroupBox" name="acc_groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1311</width>
        <height>361</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>ACC</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="gyo_groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>1311</width>
        <height>351</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>GYO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Algorithm</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1311</width>
        <height>711</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="result_label">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>180</y>
         <width>681</width>
         <height>261</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>72</pointsize>
        </font>
       </property>
       <property name="text">
        <string>井盖正常</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>261</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Algorithm Settings</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QPushButton" name="algorithm_pushButton">
        <property name="text">
         <string>Start Monitor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="calibrate_pushButton">
        <property name="text">
         <string>Calibrate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="recordButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Force Record</string>
    </property>
    <property name="shortcut">
     <string>Space</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVideo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>icons/exit.png</normaloff>icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>Port_num_comboBox</tabstop>
  <tabstop>BaudRate_lineEdit</tabstop>
  <tabstop>open_pushButton</tabstop>
  <tabstop>stop_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>onStateChanged(int)</slot>
 </slots>
</ui>
